### PR TITLE
Display flash message if wallet recovery phrase error

### PR DESCRIPTION
**Pull request by anandsinghparihar**
https://particl.atlassian.net/browse/PAR-210

### CODE REVIEW

File: src/app/modals/createwallet/createwallet.component.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { ModalsService } from '../modals.service';
 import { PassphraseComponent } from './passphrase/passphrase.component';
 import { PassphraseService } from './passphrase/passphrase.service';
 import { StateService } from '../../core/state/state.service';
+import {FlashNotificationService} from "../../services/flash-notification.service";
```
Comment (rynomster) [inline src/app/modals/createwallet/createwallet.component.ts]: this won't lint

**Comment by rynomster** [comment]
Will this flash an error message and show an error message on screen?

**Comment by anandsinghparihar** [comment]
@rynomster  Yes, flash message is displayed with an error message on screen ...  flash in disappear in some sec but screen error will be display till it not resolved.

**Comment by anandsinghparihar** [comment]
is there anything which I can do? 
> Here error will display in 1. Flash notification, 2. Error message in ModalBox

**Comment by rynomster** [comment]
Is the flash message necessary if we already have an error message on screen?

**Comment by anandsinghparihar** [comment]
hmm we can go with one error  message let me remove flash notification

**Comment by anandsinghparihar** [comment]
Please have look, I have removed flash notification.

**Pull request merged**